### PR TITLE
Refactor: align naming and config

### PR DESCRIPTION
**Pull request by axe312ger**
Preparing for a first internal test release:

* rename packages to @consent-manager/*
* rename default ui package
* delete utils
* cleanups
* fix es-module exports
* rename youTube to youtube

### CODE REVIEW

**Comment by axe312ger** [comment]
@todo there are still traces of `UnobtrusiveConsentControl`

File: package.json [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
-    "typescript": "^3.9.7"
+  "author": {
+    "name": "techboi GmbH",
+    "email": "opensource@techboi.io",
+    "url": "https://techboi.io/"
   },
-  "workspaces": [
-    "packages/*",
-    "examples/*"
+  "contributors": [
```
Comment (zcei) [inline package.json:9]: This package will never be published, wondering whether it's worth maintaining a contributors section here, as we do that for individual packages in the monorepo.

File: packages/docs/src/theme/Root.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ function Root({ children }) {
   const storage = useConsentStateStore()
   const config = {
     integrations: [
-      youTubeIntegration(),
+      youtubeIntegration(),
```
Comment (zcei) [inline packages/docs/src/theme/Root.js:31]: Looks way better 👍

File: tsconfig.json [hunk 3]
```diff
@@ -6,8 +6,6 @@
     "baseUrl": ".",
     "typeRoots": ["./node_modules/@types", "./types"],
     "paths": {
-      "@techboi/utils": ["packages/utils/src"],
-      "@techboi/react": ["packages/react/src"],
```
Comment (zcei) [inline tsconfig.json:10]: Should we add the new `@consent-manager/` packages here?

Comment (axe312ger) [reply tsconfig.json:10]: No glob support and so this will get out of sync one day. :/

File: package.json (hunk 1 above)
Comment (axe312ger) [reply package.json:9]: true that :)

**Pull request merged**